src/unittests/maths.spec.ts

```diff
@@ -94,65 +94,128 @@ g.test('test,math,diffULP')
 interface nextAfterCase {
   val: number;
   dir: boolean;
+  flush: boolean;
   result: Scalar;
 }
 
 g.test('test,math,nextAfter')
-  .paramsSimple<nextAfterCase>([
+  .paramsSubcasesOnly<nextAfterCase>([
     // Edge Cases
-    { val: Number.NaN, dir: true, result: f32Bits(0x7fffffff) },
-    { val: Number.NaN, dir: false, result: f32Bits(0x7fffffff) },
-    { val: Number.POSITIVE_INFINITY, dir: true, result: f32Bits(kBit.f32.infinity.positive) },
-    { val: Number.POSITIVE_INFINITY, dir: false, result: f32Bits(kBit.f32.infinity.positive) },
-    { val: Number.NEGATIVE_INFINITY, dir: true, result: f32Bits(kBit.f32.infinity.negative) },
-    { val: Number.NEGATIVE_INFINITY, dir: false, result: f32Bits(kBit.f32.infinity.negative) },
+    { val: Number.NaN, dir: true, flush: true, result: f32Bits(0x7fffffff) },
```

Comment:
I think it would have been cleaner to have the flush cases split into two `g.test()` blocks, but meh.

Reply:
Done